pkg/v1/providers/ytt/03_customizations/03_windows/register_antrea_cleanup.yaml

```diff
@@ -1,20 +1,19 @@
 #@ load("@ytt:overlay", "overlay")
 #@ load("@ytt:data", "data")
+
 #! TODO: This should be moved into a windows-compatible cluster class template
-#@ if data.values.IS_WINDOWS_WORKLOAD_CLUSTER:
-#@ if data.values.CNI == "antrea":
+#@ if data.values.IS_WINDOWS_WORKLOAD_CLUSTER and data.values.CNI == "antrea":
 #@overlay/match by=overlay.subset({"kind":"KubeadmConfigTemplate"}), expects="1+"
 ---
 spec:
   template:
     spec:
+      #@overlay/match missing_ok=True
       files:
```

Comment:
do I need to keep `#@overlay/append` here, it seems the default behavior though

Reply:
I am not sure. I think we should keep the `append` considering the some version of `ytt` library was replacing `files` with new content. There might be possibility that in some overlay in some other file, we already have some `files` defined and if we are not careful that can get replaced if we don't have `#@overlay/append`

So, my preference is to keep this `#@overlay/append`